ajax/libs/jquery.ime/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "jquery.ime",
+  "version": "1.0.0",
+  "description": "jQuery based internationalization library",
+  "keywords": [
+    "internationalization",
+    "localization",
+    "ime",
+    "jquery",
+    "l10n"
+  ],
+  "author": {
+    "name": "Santhosh Thottingal",
+    "email": "santhosh.thottingal@gmail.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/wikimedia/jquery.ime.git"
+  },
+  "filename": "jquery.ime.min.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/wikimedia/jquery.ime.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "jquery.ime*"
+        ]
+      },
+      {
+        "basePath": "",
+        "files": [
+          "!(README)",
+          "rules/**/*"
```

Comment:
We also need 
1. `css/jquery.ime.css`. Fetch it using the rule `css/jquery.ime*` (you can know this through .html file under `example`)
2. Because (1) is needed, all files under `images` should also be added too, use the rule `images/*`